helix-term/src/commands.rs

```diff
@@ -1188,10 +1187,11 @@ fn extend_next_long_word_end(cx: &mut Context) {
     extend_word_impl(cx, movement::move_next_long_word_end)
 }
 
-fn will_find_char<F>(cx: &mut Context, search_fn: F, inclusive: bool, extend: bool)
-where
-    F: Fn(RopeSlice, char, usize, usize, bool) -> Option<usize> + 'static,
-{
+enum SearchDirection {
```

Comment:
Rather than introduce a new type, I'd use the existing `Direction` type

https://github.com/helix-editor/helix/blob/6494fc1daf2bb379e6a1f4ec94f3dafd99b3fb34/helix-core/src/movement.rs#L23-L26

Reply:
Thanks!